source/clear-linux/tutorials/telemetry-backend/Telemetry-backend.rst

```diff
@@ -0,0 +1,495 @@
+
+Download the clearlinux/telemetrics-backend git repository
+==========================================================
+
+With all prerequisite software bundles installed and logged in with your
+administrative user, from your $HOME directory, run :command:`git` to clone
+the :guilabel:`telemetrics-backend` repository into the
+:file:`$HOME/telemetrics-backend` directory:
+
+.. code-block:: console
+
+   git clone https://github.com/clearlinux/telemetrics-backend
+
+.. note::
+   You may need to set up the :guilabel:`https_proxy` environment variable
+   if you have issues reaching github.com.
+
+Run the deploy.sh script to install the backend server
+======================================================
+
+Change your current working directory to :file:`telemetrics-backend/scripts`
+and then run the :command:`./deploy.sh -h` to see the list of options for the
+:command:`deploy.sh` script:
+
+.. code-block:: console
+
+   cd telemetrics-backend/scripts
+   ./deploy.sh -h
+  
+   Deploy snapshot of the telemetrics-backend
+
+     -a    Perform specified action (deploy, install, migrate, resetdb,
+           restart, uninstall; default: deploy)
+     -d    Distro to deploy to (ubuntu, centos or clr; default: ubuntu)
+     -h    Print these options
+     -H    Set domain for deployment (only accepted value is "localhost" for
+           now)
+     -r    Set repo location to deploy from
+           (default: https://github.com/clearlinux/telemetrics-backend)
+     -s    Set source location (default: "master" branch from git repo)
+     -t    Set source type (tarball, or git; default: git)
+     -u    Perform complete uninstallation
+
+The :command:`deploy.sh` is a bash shell script that allows you to perform the
+following actions:
+
+* deploy - install a complete instance of the telemetrics backend
+  server and all required components. This is the default action if no
+  :command:`-a` argument is given on the command line.
+* install - installs and enables all required components for the telemetrics
+  backend server.
+* migrate - migrate database to new schema.
+* resetdb - reset the database.
+* restart - restart the nginx and uWSGI services.
+* uninstall - uninstall all Ubuntu packages if the distribution is Ubuntu.
+
+For this tutorial, we will be installing the telemetrics backend server with
+the following options:
+
+* :guilabel:`-a install` to perform an install
+* :guilabel:`-d clr` to install to a |CL| distro
+* :guilabel:`-H localhost` to set the domain to localhost
+
+We do not need to set the following options since the values are set to the
+correct values we want by default:
+
+* :guilabel:`-r https://github.com/clearlinux/telemetrics-backend` sets the
+  repo location for :command:`git` to clone from.
+* :guilabel:`-s master` to set the location, or branch.
+* :guilabel:`-t git` to set the source type to git.
+
+.. caution::
+   The :command:`deploy.sh` shell script has minimal error checking and makes
```

Comment:
Since you are talking about it in the context of a file, use the file role, not the command one.
Great job adding this as a caution!

Reply:
done and thanks!